readme.md

```diff
@@ -24,38 +24,39 @@ This page is in constant WIP, please refer to it regularly.
 # The carwow Engineering Principles
 
 
-__Build what matters__ 
+__We Build what matters__ 
   
   * Engineering effort is a scarce commodity. It should only be applied to problems that “move the needle” for the company
   * The biggest achievement of an Engineer at carwow is to provide positive business impact with no line of code written (maybe even removing some)
   * We don't manage our own server, or have a self hosted continuous integration software.
     
     
-__Do *big* things in small steps__
+__We Do *big* things in small steps__
   
   * Breaking down big features in small steps doesn’t mean we work only on small incremental improvement. A longer term vision or objective help us achieve great things without sacrificing short term value.
     
     
-__Strive for fast feedback loops__
+__We Strive for fast feedback loops__
   
   * We use several practices to achieve the above like TDD, code reviews, continuous delivery, vertical slicing, AB tests, and customer feedback.
     
     
-__Investigate, Implement, Iterate__
+__We Investigate, Implement, Iterate__
   
   * We always question and challenge the value of work instead of following blindly; and then we commit to delivering early and often.
-    
 
-__Simple and clear Engineering__
+
+__We strive for Simple and clear Engineering__
   
   * We value simple and clear solutions that solve problems without unnecessary complexity. At the same time we keep the long term vision in mind and don’t just build for today. 
     
 
-__Outside in ownership__
+__We employ Outside-In ownership__
   
-  * We value shipping early and observing in production; we are fearless. We believe a developer is responsible for owning their work all the way through to delivering value, including monitoring and performance. 
+  * We value shipping early and observing in production.
+  * We are Fearless; We believe a developer is responsible for owning their work all the way through to delivering value, including monitoring and performance. 
```

Comment:
Fearless is interesting, does that come from any specific blogpost/manifesto that's worth linking to? I agree with the idea, but I normally define this as `We value ownership`, maybe fearless ownership?

Reply:
It was in the original one, I thought it was interesting too and was too buried so just called it out seperately

Reply:
I like pointing out ownership/responsibility.
I’m not clear on how that amounts to fearlessness though - could you clarify @vrashabhirde?

Reply:
I think the fearless part was meant to the previous point:
`We value shipping early and observing in production. We are Fearless.`

Reply:
I think having it in the previous section is better, now the first point is less memorable?

Reply:
I like the word "Fearless" here. Although I think there's an overlap with `Confident` - which is one of the carwow core values.

Reply:
I would like to suggest a move away from fearless as it can be seen as uninclusive. Could we reword this towards how we tackle fear?

```
We fight fear; we believe in developer ownership and responsibility within a supportive, inclusive support network that allows them to deliver amazing value and grow themselves.
``` 

Or written better by someone else. I think this way we show ourselves as friendly to junior's and senior's and people who have struggled from being dropped in the deep end. Because Carwow doesn't do that, it fosters and supports its employees and we should show that in this document.

Reply:
I prefer that suggestion, it has the same sentiment while being more inclusive. Win-Win

Reply:
I relate the fearlessness more with shipping early and often, rather than with ownership and responsibility. Like what @PChambino said

i.e. 
- we are not afraid to try and test things.
- we believe in ownership and responsibility within a supportive framework
(in more eloquent words maybe)